data/web/sogo-auth.php

```diff
@@ -1,62 +1,63 @@
 <?php
+require_once $_SERVER['DOCUMENT_ROOT'] . '/inc/prerequisites.inc.php';
```

Comment:
I specifically added it only on requests with $_GET['login'], so it is not executed on auth_request calls, but I don't know how much cpu time the prerequisites consume, i'll leave the decision to you :)

Reply:
maybe it was not the best idea in the first place to use the same file for redirection and auth_request

Reply:
I should ask before changing your code... sorry.